sky/server/config.py

```diff
@@ -225,7 +227,7 @@ def _get_min_short_workers() -> int:
     return _MIN_IDLE_SHORT_WORKERS + daemon_count
 
 
-def _max_short_worker_parallism(mem_size_gb: float,
+def _max_short_worker_parallism(cpu_count: int, mem_size_gb: float,
                                 long_worker_parallism: int) -> int:
```

Comment:
Not part of your PR but we should fix
```suggestion
                                long_worker_parallelism: int) -> int:
```